redisinsight/api/src/__mocks__/cloud-subscription.ts

```diff
@@ -67,3 +71,111 @@ export const mockCloudSubscriptionFixed = Object.assign(new CloudSubscription(),
   type: CloudSubscriptionType.Fixed,
   price: mockCloudCapiSubscriptionFixed.price,
 });
+
+export const mockCloudCapiCloudRegion1: ICloudCapiSubscriptionCloudRegion = {
+  id: '1',
+  region_id: 1,
+  zone_id: null,
+  name: 'us-east-1',
+  cloud: 'aws',
+  support_maz: true,
+  country_name: 'USA',
+  city_name: 'Vegas',
+  flag: 'fr',
+  longitude: null,
+  latitude: null,
+  display_order: 1,
+};
+
+export const mockCloudCapiCloudRegion2: ICloudCapiSubscriptionCloudRegion = {
+  id: '2',
+  region_id: 2,
+  zone_id: null,
+  name: 'asia-northeast2',
+  cloud: 'gpc',
+  support_maz: true,
+  country_name: 'Poland',
+  city_name: 'Warsaw',
+  flag: 'pl',
+  longitude: null,
+  latitude: null,
+  display_order: 2,
+};
+
+export const mockCloudCapiCloudRegions: ICloudCapiSubscriptionCloudRegion[] = [
+  mockCloudCapiCloudRegion1,
+  mockCloudCapiCloudRegion2,
+];
+
+export const mockFreeCloudSubscriptionPlan1: CloudSubscriptionPlan = {
```

Comment:
I don't see region_id here. we must connect by this field

Reply:
added